src/main.py

```diff
@@ -13,13 +13,6 @@
 import signal, sys, itertools
 import math
 
-def signal_handler(sig, frame):
-    tournament.close()
```

Comment:
Had to get rid of this because whenever the user sends a SIGINT (ctl-c) in the middle of a tournament, `.close()` will save the models. But we only want to do this at the end of the round to make sure that all our strategies have the same amount of games played